sdk/nodejs/client.ts

```diff
@@ -22,6 +22,9 @@ export class Client {
 
 	/**
 	 * creates a new Dagger Typescript SDK GraphQL client.
+	 * If the client is created by `dagger.connect()`, it will
+	 * hold the serverProcess, so it can be closed using `close()`
+	 * method.
```

Comment:
No longer accurate. Remove this comment.

Reply:
Done. Thanks